horovod/common/operations.cc

```diff
@@ -1094,7 +1096,7 @@ void PerformOperation(TensorTable& tensor_table, MPIResponse response) {
       delete[] recvcounts;
       delete[] displcmnts;
 
-      for (size_t ec = 0; ec < entries.size(); ec++) {
+      for (size_t ec = 0; ec < entries.size(); ++ec {
```

Comment:
Missing closing parenthesis.  Please compile the code locally before submitting.